mix/CodeModel.cpp

```diff
@@ -279,9 +279,22 @@ void CodeModel::runCompilationJob(int _jobId)
 				QQmlEngine::setObjectOwnership(contract, QQmlEngine::CppOwnership);
```

Comment:
Quite long function. Does it make sense to create a new function where the guards start their work?